packages/Notification/__tests__/Notification.spec.jsx

```diff
@@ -135,4 +135,27 @@ describe('<Notification />', () => {
       expect(icon.props().variant).toBeUndefined()
     })
   })
+  describe('onDismiss', () => {
```

Comment:
Can we add tests for `onExit`?

Reply:
We can't due to the difficulty of testing things using transition group, and also because we're getting into territory of testing the functionality of react.